### DISCUSSION TITLE
How do I display all values ​​on the x axis?

### DISCUSSION THREAD

**Discussion by popkovanton** [patrykandpatrick/vico#815] [Questions]
### Question

I used the first graph from the sample as an example. As you can see in the screenshot, the x-axis value labels are drawn with a gap. 
How to display all values, with the same data set?

```
@Composable
private fun ComposeChart1(modelProducer: CartesianChartModelProducer, modifier: Modifier) {
  val marker = rememberMarker()
  CartesianChartHost(
    chart =
      rememberCartesianChart(
        rememberLineCartesianLayer(
          LineCartesianLayer.LineProvider.series(
            rememberLine(DynamicShader.color(Color(0xffa485e0)))
          ),
          axisValueOverrider = AxisValueOverrider.fixed(
            minX = 0.0,
            maxX = 10.0,
          ),
        ),
        startAxis = rememberStartAxis(),
        bottomAxis = rememberBottomAxis(guideline = null, tick = null),
      ),
    modelProducer = modelProducer,
    modifier = modifier,
  )
}

private val x = listOf(6, 9)
```

### Vico version(s)

2.0.0-alpha.25

### UI framework(s)

Jetpack Compose

**Comment by popkovanton** [comment]
![Screenshot_20240726_203526](https://github.com/user-attachments/assets/e011cd7c-66df-4966-90fa-cd7453c3cf61)

**Comment by patrickmichalik** [comment]
Hello! You can do this by setting the _x_ step to 1 via `rememberCartesianChart`’s `getXStep` parameter. The default _x_ step is the GCD of the differences between neighboring _x_ values, which is 3 in this case.
